### ISSUE TITLE
Release plan for 2.3.0?

### ISSUE THREAD

**Issue by lcts** [h2o/h2o#2317]
Is there some kind of roadmap for a 2.3.0 stable release? FWIW, I've been using the beta since November and had no issues.

**Comment by xm74** [comment]
I'm ok even with new 2.3.0-beta3 because a lot of time is passed from previous one.

**Comment by dch** [comment]
2.3.0-b2 tag was added on 2019-08-13 @kazuho if we are waiting on a specific feature (e.g. http/3) then it would be can at least help set some expectations

**Comment by Totktonada** [comment]
Sorry for disturbing you, but I would gently remind about the question. Are there plan, checklist or something else that would give us some expectations about future releases?

**Comment by utrenkner** [comment]
I am only a user, but this is my understanding of the project: The development has been taken over almost completely by Fastly. They use it in their CDN offerings. H2O as a stand-alone webserver is not high on their priority list. I would understand this and am thankful to Fastly for keeping this repo up-to-date and free (MIT licensed).

Furthermore, I recommend to everyone to use the latest code from the Master branch and to report any problems back to the developers. This way we can help improve this project. Thanks to @dch 's latest update of the FreeBSD h2o-devel port, I now run a recent snapshot on several production servers - without any problems at all.

Still, I hope that now that HTTP/3 final has landed, there is a chance to take a little break from the rapid development and branch a new release (which could be 3.0 as a tribute to its integrating of HTTP/3)...  Just dreaming.

**Comment by CarloMartini** [comment]
I also use h2o in production (multiple servers with many different services): quite happy about it, but after two years since the release of 2.2.6, a roadmap about further development would be nice to have.
